src/components/assets/EvmAssetList.vue

```diff
@@ -71,7 +71,7 @@
             <div class="column column--balance">
               <div class="column__box">
                 <div class="text--accent">
-                  <TokenBalance :balance="bal" :symbol="nativeTokenSymbol" />
+                  <TokenBalance :balance="String(bal)" :symbol="nativeTokenSymbol" />
```

Comment:
Instead having String conversion on each TokenBalance component call, can you move this conversion inside the component. So pass number to the component and inside component do String(balance)

Reply:
@bobo-k2 
No, Vue will display the warning messages whenever passing the wrong format as props to components. 
Actually, what I want is to set the props type as `type: String || Number` like [this](https://github.com/AstarNetwork/astar-apps/pull/538/files#diff-1081888f2d78afd6f064737cb0b785e8ad4c26ca472179d8b973728533745c33L18) but sadly it is not working.


<img width="542" alt="image" src="https://user-images.githubusercontent.com/92044428/196718863-64afb5bd-add8-4e14-b8f5-5126b139f7a8.png">

Reply:
Can you try with type: [String, Number]

Reply:
Thank you, but I still got this error...

<img width="546" alt="image" src="https://user-images.githubusercontent.com/92044428/196728050-fa69be34-9f9a-469d-b6c2-4cc70a4a1246.png">

<img width="546" alt="image" src="https://user-images.githubusercontent.com/92044428/196728336-04dd6f84-0ed9-4a5e-89a1-fcd5596aba06.png">